virus/Totivirus_Arthropodamyoti.md

```diff
@@ -0,0 +1,77 @@
+
+From the other observations of the assigned sOTU, I found other SRRs containing the sOTU which were from projects 'PRJNA453114', 'PRJNA835632', 'PRJNA834940', 'PRJNA693645' involving organisms: Myotis myotis, Calliphora vicina, Phumosia chukanella, Stegodyphus africanus. The Stegodyphus africanus were from South Africa, Myotis myotis were from Sweden, Calliphora vicina were purchased from suppliers in Germany. The publication associated with SRR19115554 is a study of the virome of bat colonies in Sweden [Wiederkehr 2022]. 
+
+Overall, most of the publications associated with my findings from BLAST and the SRR accessions are studies of the viruses in potential hosts/reservoirs. The common associated host class appears to be bat species and the vector seems to be arthropods such as flies. 
+
+Upon filtering the 'virusRdRPCandidates' file for the barcode 'u508736', it resulted in 2087 matches with an A, B and C motif detected. The highest coverage (144497.99) was 'NODE_21893'. Then, I looked into the transcript using BLAST using the 'Somewhat similar sequences' parameter as there were no significant hits with 'megablast'. There were four hits in this search for similar sequences in which three were from the Totiviridae sp. (MZ218615.1,KY024325.1, MZ218620.1) and one was from Biomphalaria virus 5 (KY024325.1). The associated publications from these Totiviridae hits used environmental samples (animal faecal samles on terrestrial sites) across China. The Biomphalaria virus 5 was found in freshwater snails from the genus Biomphalaria which is a host of the parasite Schistosoma mansoni.
+
+### Virus Genome {Q3}
+
+![Genome map](img/Totivirus_Arthropodamyoti/genome_map.png)
+>Figure 1. Genome map of Totivirus Arthropodamyoti. 
```

Comment:
Could you extend all your Figure Legends a little bit with more of a description of what you're showing.